cmeutils/tests/test_structure.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 import freud
 
-from cmeutils.tests.base_test import BaseTest
```

Comment:
Were the tests not working for you without this change? It might be because cmeutils is not installed in your environment or not installed in an editable way? (i.e., `pip install -e .`) Not sure.